admin_guide/out_of_resource_handling.adoc

```diff
@@ -64,8 +68,7 @@ Each threshold is of the following form:
 <eviction_signal><operator><quantity>
 ----
 
-* Valid `eviction-signal` tokens as defined in
-xref:out-of-resource-eviction-signals[Eviction Signals].
+* Valid `eviction-signal` tokens as defined by eviction signals.
 * Valid `operator` tokens are `<`.
 * Valid `quantity` tokens must match the quantity representation used by
 Kubernetes.
```

Comment:
this basically means 'the syntax you used to express a quantity anywhere in openshift/kube' - so whether its how you declare the amount of cpu or memory for a pod, a constraint on quota, a limit in a limit range, etc.  they all use the quantity representation.  maybe its just obvious.  we don't appear to have a good doc to describe: 
https://github.com/kubernetes/kubernetes/blob/master/docs/design/resources.md#resource-quantities